src/env.c

```diff
@@ -1,6 +1,26 @@
 #include "rlang.h"
 
+
+bool r_is_unbound_value(SEXP x) {
+  return x == R_UnboundValue;
+}
+
 SEXP r_mut_env_parent(SEXP env, SEXP new_parent) {
```

Comment:
> because it seems to imply that sexp are otherwise const

on hindsight maybe not: we can just see it as a "restating the obvious" keyword. It doesn't change the status of the argument since it's mutable by default, but stresses that this is a case where we do want it mutable, which is what we want for documentation purposes.